services/vidispine/chapter_test.go

```diff
@@ -53,38 +53,67 @@ func TestMergeTerseTimecodes(t *testing.T) {
 	assert.Equal(t, expected, result)
 }
 
-func Test_GetChapterMetaForClips(t *testing.T) {
-	clips := []*Clip{
+// This is an edge case where the annotations overlap by a few frames
+func Test_GetChapterMetaForClips_Overlapping(t *testing.T) {
+	if os.Getenv("VIDISPINE_BASE_URL") == "" {
+		t.Skip("VIDISPINE_BASE_URL is not set")
+	}
+
+	testData := [][]*Clip{
+		{
+			{
+				VideoFile:   "/dummy/file.mp4",
+				InSeconds:   1420.1199999999953,
+				OutSeconds:  2767.439999999988,
+				SequenceIn:  0,
+				SequenceOut: 0,
+				AudioFiles: map[string]*AudioFile{
+					"nor": {
+						VXID:    "VX-489605",
+						Streams: []int{2},
+						File:    "/dummy/file.wav",
+					},
+				},
+				SubtitleFiles:      map[string]string{},
+				JSONTranscriptFile: "",
+				VXID:               "VX-489598",
+			},
+		},
 		{
-			VideoFile:   "/dummy/file.mp4",
-			InSeconds:   1907.7599999999948,
-			OutSeconds:  3025.399999999994,
-			SequenceIn:  0,
-			SequenceOut: 0,
-			AudioFiles: map[string]*AudioFile{
-				"nor": {
-					VXID:    "VX-486737",
-					Streams: []int{1, 2},
-					File:    "/dummy/file.wav",
+			{
+				VideoFile:   "/dummy/file.mp4",
+				InSeconds:   3750.9199999999983,
+				OutSeconds:  3906.87999999999,
+				SequenceIn:  0,
+				SequenceOut: 0,
+				AudioFiles: map[string]*AudioFile{
+					"nor": {
+						VXID:    "VX-489605",
+						Streams: []int{2},
+						File:    "/dummy/file.wav",
+					},
 				},
+				SubtitleFiles:      map[string]string{},
+				JSONTranscriptFile: "",
+				VXID:               "VX-489598",
 			},
-			SubtitleFiles:      map[string]string{},
-			JSONTranscriptFile: "",
-			VXID:               "VX-486737",
```

Comment:
lost this id?

Reply:
maybe document that the IDs here actually matter, and maybe how those assets can be recreated if lost

Reply:
^ in general for all these kinds of tests if we have more of them